### PR TITLE
Maintenance changes

### PR DESCRIPTION
**Pull request by arb**
Updated .travis to include new versions of node. Updated depenencies, moved to
Map() where it made sence to do so.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *
-!lib/**
\ No newline at end of file
+!lib/**
```
Comment (cjihrig) [inline .npmignore]: It doesn't look like there is a `lib` directory in this project.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
-  - "6"
+    - "4"
```
Comment (cjihrig) [inline .travis.yml]: Why the increase in indentation? What about Node 9?

Comment (arb) [reply .travis.yml]: Just copying what I've seen elsewhere

File: .npmignore (hunk 1 above)
Comment (arb) [reply .npmignore]: I guess that's true.

**Pull request merged**